bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoCrockpotHandler.java

```diff
@@ -0,0 +1,307 @@
+        String actionService = "basicevent";
+
+        String soapHeader = "\"urn:Belkin:service:" + actionService + ":1#" + action + "\"";
+        String content = "<?xml version=\"1.0\"?>"
+                + "<s:Envelope xmlns:s=\"http://schemas.xmlsoap.org/soap/envelope/\" s:encodingStyle=\"http://schemas.xmlsoap.org/soap/encoding/\">"
+                + "<s:Body>" + "<u:" + action + " xmlns:u=\"urn:Belkin:service:" + actionService + ":1\">" + "</u:"
+                + action + ">" + "</s:Body>" + "</s:Envelope>";
+
+        try {
+            String wemoURL = getWemoURL(actionService);
+            if (wemoURL != null) {
+                String wemoCallResponse = wemoHttpCaller.executeCall(wemoURL, soapHeader, content);
+                if (wemoCallResponse != null) {
+                    logger.trace("State response '{}' for device '{}' received", wemoCallResponse, getThing().getUID());
+                    String mode = StringUtils.substringBetween(wemoCallResponse, "<mode>", "</mode>");
+                    String time = StringUtils.substringBetween(wemoCallResponse, "<time>", "</time>");
+                    String coockedTime = StringUtils.substringBetween(wemoCallResponse, "<coockedTime>",
+                            "</coockedTime>");
+
+                    if (mode != null && time != null && coockedTime != null) {
+                        State newMode = new StringType(mode);
+                        State newCoockedTime = DecimalType.valueOf(coockedTime);
+                        switch (mode) {
+                            case "0":
+                                newMode = new StringType("OFF");
+                                break;
+                            case "50":
+                                newMode = new StringType("WARM");
+                                State warmTime = DecimalType.valueOf(time);
+                                updateState(CHANNEL_WARMCOOKTIME, warmTime);
+                                break;
+                            case "51":
+                                newMode = new StringType("LOW");
+                                State lowTime = DecimalType.valueOf(time);
+                                updateState(CHANNEL_LOWCOOKTIME, lowTime);
+                                break;
+                            case "52":
+                                newMode = new StringType("HIGH");
+                                State highTime = DecimalType.valueOf(time);
+                                updateState(CHANNEL_HIGHCOOKTIME, highTime);
+                                break;
+                        }
+                        updateState(CHANNEL_COOKMODE, newMode);
+                        updateState(CHANNEL_COOKEDTIME, newCoockedTime);
+                    }
+                }
+            }
+        } catch (RuntimeException e) {
+            logger.debug("Failed to get actual state for device '{}': {}", getThing().getUID(), e.getMessage());
```

Comment:
Also log the exception itself in case of a RuntimeException to know where it originated.
```suggestion
            logger.debug("Failed to get actual state for device '{}': {}", getThing().getUID(), e.getMessage(), e);
```